roles/pytools/tasks/main.yml

```diff
@@ -12,7 +12,8 @@
 
   - name: Create virtualenv
     pip:
-      name: git+https://github.com/stackhpc/slurm-openstack-tools.git
+      name: "git+https://github.com/stackhpc/slurm-openstack-tools.git{{ '#egg=slurm_openstack_tools' if pytools_editable else ''}}"
```

Comment:
```suggestion
      name: "git+https://github.com/stackhpc/slurm-openstack-tools.git#egg=slurm_openstack_tools"
```
Think we can set this unconditionally, right? See: https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support.